test/playback.js

```diff
@@ -215,4 +253,46 @@ describe('A Google Music instance', function () {
       });
     });
   });
+
+  describe('when we set repeat to repeat list', function () {
+    browserUtils.execute(function setToRepeatList () {
+      window.gmusic.playback.setRepeat(window.GMusic.Playback.LIST_REPEAT);
+    });
+    describe('via API call', function () {
+      browserUtils.execute(function getRepeatMode () {
+        return window.gmusic.playback.getRepeat();
+      });
+      it('was set to repeat list', function () {
+        expect(this.result).to.be.equal('LIST_REPEAT');
+      });
+    });
+  });
+
+  describe('when we set repeat to repeat one', function () {
+    browserUtils.execute(function setToRepeatOne () {
+      window.gmusic.playback.setRepeat(window.GMusic.Playback.SINGLE_REPEAT);
+    });
+    describe('via API call', function () {
```

Comment:
We don't need to nest any of this via a describe block. `mocha` supports running successive befores in order (`browserUtils.execute` is running `before` under the hood)